src/Spinners/LoadSpinnerText.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+export interface LoadSpinnerTextProps {
+  /** Specifies the text under the spinner */
+  children: string;
+  /** Specifies the color of the text under the spinner */
+  color?: string;
+}
+
+const StyledLoadSpinnerText = styled.div`
+  background: transparent;
+  color: ${({ color }): string => (color)};
+  font: ${({ theme }): string => (theme.font.heading)};
```

Comment:
I noted this before, but I think it got lost in the refactoring:

You can't actually use the font values in our theme this way. You have to specific the family, size, and weight separately, with:

```tsx
   font-family: ${({ theme }): string => (theme.font.heading.family)};
   font-weight: ${({ theme }): string => (theme.font.heading.weight)};
   font-size: ${({ theme }): string => (theme.font.heading.size)};
```